### ISSUE TITLE
Smaller minimal size columns for numbers and artwork

### ISSUE THREAD

**Issue by PanderMusubi** [nightingale-media-player/nightingale-hacking#195]
Please use a smaller minimal size for columns with numbers such as track number, total tracks, disc number, total discs and for artwork. The width of the artwork in the artwork column could be used as a minimum for all these columns.

Implementing this allows the user to optimise the space used by columns to get maximum usage of the available width of the overview of songs.

**Comment by PanderMusubi** [comment]
Nope. Meanwhile I cannot re-enable showing album art in display view via changing some setting in about:config. Could you tell me again what setting I need to change to get that available again to show album art as a column?

**Comment by PanderMusubi** [comment]
Ah, found it again. Was songbird.columnpicker.allowSecondaryProperties true

**Comment by PanderMusubi** [comment]
To come back to the original question, is it possible to force the minimal width for column Artwork to the height of the cells? Or when renaming the name of the column to "Art", it will save with because no dots are needed as is the case in the current minimal width. Then you see "Art...".

I don't get what you mean with feather specific. This proposed fix relates to the default main pane for Library / Music.

**Comment by freaktechnik** [comment]
I _think_ the columns of the playlist widget are just reskinned GTK+ columns, and I don't think any min width gets passed in. But the definition for that would probably be in playlist.xml.

**Comment by PanderMusubi** [comment]
Where do I find that file? Can't find in the installed .deb for Ubuntu for 1.13a

**Comment by darealshinji** [comment]
It's probably inside one of the .jar files then.

**Comment by freaktechnik** [comment]
Yep, it's inside the songbird.jar... It's probably easier to look at here on gihub, for example. This is where the actual column headers get generated: [app/content/bindings/playlist.xml#L1330-L1392](https://github.com/nightingale-media-player/nightingale-hacking/blob/sb-trunk-oldxul/app/content/bindings/playlist.xml#L1330-L1392). ~~To achieve the wanted effect, the treecols elements probably need the `crop="right"` attribute, as documented [here](https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XUL/treecol).~~

EDIT: sorry, hit the wrong button
EDIT2: oops, crop is just visual.

**Comment by PanderMusubi** [comment]
At the moment "Track #" stands for "Track Number" to save width for the column. Please also do that for "Disc Number" and use the label "Disc #" for the column.
